scripts/jenkins/ardana/heat-ardana-deployerincloud-lite.yaml

```diff
@@ -183,7 +183,7 @@ resources:
 
 outputs:
   # deployer-controller
-  deployer-controller-ip-floating:
+  deployer-ip-floating:
```

Comment:
How is that change enabling `std-min`?

Reply:
@nicolasbock I updated the commit message reflecting that.